From 25525b7f7232863e191d4806ed8c0b40a1919456 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Tue, 12 Mar 2024 12:37:48 +0800
Subject: [PATCH 7/8] chore: use the wheel group for hardware access

This makes it easier to make VirtualBox work out-of-the-box.
---
 debian/postinst                               |  4 ++--
 debian/templates                              |  6 ++---
 doc/manual/en_US/user_BasicConcepts.xml       |  2 +-
 doc/manual/en_US/user_Installation.xml        | 22 +++++++++----------
 doc/manual/en_US/user_Security.xml            |  2 +-
 doc/manual/en_US/user_Troubleshooting.xml     |  2 +-
 .../Frontends/VirtualBox/nls/VirtualBox_bg.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_ca.ts |  4 ++--
 .../VirtualBox/nls/VirtualBox_ca_VA.ts        |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_cs.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_da.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_de.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_el.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_es.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_eu.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_fa.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_fi.ts |  2 +-
 .../Frontends/VirtualBox/nls/VirtualBox_fr.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_gl.ts |  2 +-
 .../Frontends/VirtualBox/nls/VirtualBox_he.ts |  2 +-
 .../VirtualBox/nls/VirtualBox_hr_HR.ts        |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_hu.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_id.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_it.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_ja.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_ka.ts |  2 +-
 .../VirtualBox/nls/VirtualBox_km_KH.ts        |  2 +-
 .../Frontends/VirtualBox/nls/VirtualBox_ko.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_lt.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_nl.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_pl.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_pt.ts |  2 +-
 .../VirtualBox/nls/VirtualBox_pt_BR.ts        |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_ro.ts |  2 +-
 .../Frontends/VirtualBox/nls/VirtualBox_ru.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_sk.ts |  2 +-
 .../Frontends/VirtualBox/nls/VirtualBox_sl.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_sr.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_sv.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_th.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_tr.ts |  4 ++--
 .../Frontends/VirtualBox/nls/VirtualBox_uk.ts |  4 ++--
 .../VirtualBox/nls/VirtualBox_xx_YY.ts        |  2 +-
 .../VirtualBox/nls/VirtualBox_zh_CN.ts        |  4 ++--
 .../VirtualBox/nls/VirtualBox_zh_TW.ts        |  4 ++--
 .../src/globals/UIMessageCenter.cpp           |  2 +-
 src/VBox/HostDrivers/linux/loadall.sh         |  6 ++---
 src/VBox/Installer/linux/VBoxCreateUSBNode.sh |  2 +-
 src/VBox/Installer/linux/debian/postinst      |  4 ++--
 src/VBox/Installer/linux/debian/templates     |  6 ++---
 src/VBox/Installer/linux/install.sh           |  2 +-
 .../Installer/linux/rpm/VirtualBox.tmpl.spec  |  4 ++--
 src/VBox/Installer/linux/vboxdrv.sh           |  4 ++--
 src/VBox/Installer/solaris/vbox-ips.mog       |  4 ++--
 src/VBox/Main/nls/VirtualBoxAPI_ru.ts         |  4 ++--
 src/VBox/Main/nls/VirtualBoxAPI_tr.ts         |  4 ++--
 src/VBox/Main/nls/VirtualBoxAPI_xx_YY.ts      |  2 +-
 src/VBox/Main/src-client/ConsoleImpl2.cpp     |  2 +-
 src/VBox/Main/src-server/HostImpl.cpp         |  2 +-
 59 files changed, 112 insertions(+), 112 deletions(-)

diff --git a/debian/postinst b/debian/postinst
index 197ede68..bc789b5a 100755
--- a/debian/postinst
+++ b/debian/postinst
@@ -59,9 +59,9 @@ if [ "$1" = "configure" ]; then
 
   # create users groups (disable with INSTALL_NO_GROUP=1 in /etc/default/virtualbox)
   if [ "$INSTALL_NO_GROUP" != "1" ]; then
-    db_input low virtualbox/group-vboxusers || true
+    db_input low virtualbox/group-wheel || true
     db_go || true
-    addgroup --system vboxusers || true
+    addgroup --system wheel || true
   fi
 
   # The starters need to be Suid root. They drop the privileges before starting
diff --git a/debian/templates b/debian/templates
index 341644b5..ec12350b 100644
--- a/debian/templates
+++ b/debian/templates
@@ -1,9 +1,9 @@
-Template: virtualbox/group-vboxusers
+Template: virtualbox/group-wheel
 Type: note
-Description: Creating group 'vboxusers'
+Description: Creating group 'wheel'
  Users of VirtualBox must be members of this group. Host network interfaces
  will be assigned to this group.
-Description-de.UTF-8: Erzeuge Gruppe 'vboxusers'
+Description-de.UTF-8: Erzeuge Gruppe 'wheel'
  Nutzer von VirtualBox müssen Mitglied dieser Gruppe sein. Hostinterfaces
  werden dieser Gruppe zugewiesen.
 
diff --git a/doc/manual/en_US/user_BasicConcepts.xml b/doc/manual/en_US/user_BasicConcepts.xml
index 285072e2..5b8b1db6 100644
--- a/doc/manual/en_US/user_BasicConcepts.xml
+++ b/doc/manual/en_US/user_BasicConcepts.xml
@@ -2479,7 +2479,7 @@
         On supported Linux hosts, &product-name; accesses USB devices
         through special files in the file system. When &product-name; is
         installed, these are made available to all users in the
-        <literal>vboxusers</literal> system group. In order to be able
+        <literal>wheel</literal> system group. In order to be able
         to access USB from guest systems, make sure that you are a
         member of this group.
       </para>
diff --git a/doc/manual/en_US/user_Installation.xml b/doc/manual/en_US/user_Installation.xml
index 2661b259..091ea59a 100644
--- a/doc/manual/en_US/user_Installation.xml
+++ b/doc/manual/en_US/user_Installation.xml
@@ -824,7 +824,7 @@
           <listitem>
             <para>
               Creates a new system group called
-              <literal>vboxusers</literal>.
+              <literal>wheel</literal>.
             </para>
           </listitem>
 
@@ -846,7 +846,7 @@
               <filename>/etc/udev/rules.d/60-vboxdrv.rules</filename>, a
               description file for udev, if that is present, which makes
               the USB devices accessible to all users in the
-              <literal>vboxusers</literal> group.
+              <literal>wheel</literal> group.
             </para>
           </listitem>
 
@@ -876,12 +876,12 @@
 
         <para>
           Add every user who needs to access USB devices from a
-          VirtualBox guests to the group <literal>vboxusers</literal>.
+          VirtualBox guests to the group <literal>wheel</literal>.
           Either use the OS user management tools or run the following
           command as root:
         </para>
 
-<screen>sudo usermod -a -G vboxusers username</screen>
+<screen>sudo usermod -a -G wheel username</screen>
 
         <note>
           <para>
@@ -893,7 +893,7 @@
             <command>groups</command> command and add all these groups
             in a comma-separated list to the command line after the
             <option>-G</option> option. For example: <command>usermod -G
-            <replaceable>group1</replaceable>,<replaceable>group2</replaceable>,vboxusers
+            <replaceable>group1</replaceable>,<replaceable>group2</replaceable>,wheel
             <replaceable>username</replaceable></command>.
           </para>
         </note>
@@ -1107,7 +1107,7 @@ virtualbox virtualbox/delete-old-modules boolean true</screen>
 <screen>INSTALL_NO_UDEV=1</screen>
 
         <para>
-          The creation of the group vboxusers can be prevented as
+          The creation of the group wheel can be prevented as
           follows:
         </para>
 
@@ -1125,20 +1125,20 @@ virtualbox virtualbox/delete-old-modules boolean true</screen>
 
     </sect2>
 
-    <sect2 id="install-linux-vboxusers">
+    <sect2 id="install-linux-wheel">
 
-      <title>The vboxusers Group</title>
+      <title>The wheel Group</title>
 
       <para>
         The Linux installers create the system user group
-        <literal>vboxusers</literal> during installation. Any system
+        <literal>wheel</literal> during installation. Any system
         user who is going to use USB devices from &product-name; guests
         must be a member of that group. A user can be made a member of
-        the group <literal>vboxusers</literal> either by using the
+        the group <literal>wheel</literal> either by using the
         desktop user and group tools, or with the following command:
       </para>
 
-<screen>sudo usermod -a -G vboxusers username</screen>
+<screen>sudo usermod -a -G wheel username</screen>
 
     </sect2>
 
diff --git a/doc/manual/en_US/user_Security.xml b/doc/manual/en_US/user_Security.xml
index 6c3b986e..f8edfc69 100644
--- a/doc/manual/en_US/user_Security.xml
+++ b/doc/manual/en_US/user_Security.xml
@@ -173,7 +173,7 @@
         and Linux hosts it is necessary to configure the proper
         permissions for users executing VMs and who should be able to
         access certain host resources. For instance, Linux users must be
-        member of the <emphasis>vboxusers</emphasis> group to be able to
+        member of the <emphasis>wheel</emphasis> group to be able to
         pass USB devices to a guest. If a serial host interface should
         be accessed from a VM, the proper permissions must be granted to
         the user to be able to access that device. The same applies to
diff --git a/doc/manual/en_US/user_Troubleshooting.xml b/doc/manual/en_US/user_Troubleshooting.xml
index 4906fd48..c3df67f5 100644
--- a/doc/manual/en_US/user_Troubleshooting.xml
+++ b/doc/manual/en_US/user_Troubleshooting.xml
@@ -1863,7 +1863,7 @@ VirtualBox</screen>
 
       <para>
         If USB is not working on your Linux host, make sure that the
-        current user is a member of the <literal>vboxusers</literal>
+        current user is a member of the <literal>wheel</literal>
         group. Please keep in mind that group membership does not take
         effect immediately but rather at the next login. If available,
         the <command>newgrp</command> command may avoid the need for a
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_bg.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_bg.ts
index 869cf8e3..6be6143d 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_bg.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_bg.ts
@@ -16957,8 +16957,8 @@ p, li { white-space: pre-wrap; }
         <translation>Не може да се зареди услугата Host USB Proxy (VERR_FILE_NOT_FOUND). Услугата може да не е инсталирана на реалния компютър</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>Текущо не е позволено VirtualBox да достъпва USB устройства. Можете да промените това с добавяне на вашия потребител към групата &apos;vboxusers&apos;.  Моля, погледнете потребителското ръководство за по-подробно описание</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>Текущо не е позволено VirtualBox да достъпва USB устройства. Можете да промените това с добавяне на вашия потребител към групата &apos;wheel&apos;.  Моля, погледнете потребителското ръководство за по-подробно описание</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ca.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ca.ts
index 4b2036f1..3cc721c1 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ca.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ca.ts
@@ -11688,8 +11688,8 @@
         <translation>No s&apos;ha pogut carregar el servei de servidor intermediari USB (VERR_FILE_NOT_FOUND). Pot ser que el servei no estigui instal·lat a l&apos;ordinador amfitrió</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>El VirtualBox no té permisos per a accedir als dispositius USB. Podeu canviar això afegint el vostre usuari al grup «vboxusers». Vegeu el manual de l&apos;usuari per a una explicació més detallada</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>El VirtualBox no té permisos per a accedir als dispositius USB. Podeu canviar això afegint el vostre usuari al grup «wheel». Vegeu el manual de l&apos;usuari per a una explicació més detallada</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ca_VA.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ca_VA.ts
index f6724ba3..920399fe 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ca_VA.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ca_VA.ts
@@ -16178,8 +16178,8 @@ p, li { white-space: pre-wrap; }
         <translation>No s&apos;ha pogut carregar el servei de servidor intermediari USB (VERR_FILE_NOT_FOUND). Pot ser que el servei no estiga instal·lat a l&apos;ordinador amfitrió</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>El VirtualBox no té permisos per accedir als dispositius USB. Podeu canviar això afegint el vostre usuari al grup «vboxusers». Vegeu el manual de l&apos;usuari per a una explicació més detallada</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>El VirtualBox no té permisos per accedir als dispositius USB. Podeu canviar això afegint el vostre usuari al grup «wheel». Vegeu el manual de l&apos;usuari per a una explicació més detallada</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_cs.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_cs.ts
index 93160f86..a0204b8e 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_cs.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_cs.ts
@@ -16994,8 +16994,8 @@ p, li { white-space: pre-wrap; }
         <translation>Nelze spustit službu USB Proxy (VERR_FILE_NOT_FOUND). Zřejmě na tomto hostutelském počítači není služba nainstalována</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>Aplikace VirtualBox nemůže nyní přistupovat k USB zařízením. Můžete to změnit přidáním uživatele do skupiny &apos;vboxusers&apos;. Více informací najdete v uživatelské příručce</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>Aplikace VirtualBox nemůže nyní přistupovat k USB zařízením. Můžete to změnit přidáním uživatele do skupiny &apos;wheel&apos;. Více informací najdete v uživatelské příručce</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_da.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_da.ts
index 112ded24..b75904d5 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_da.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_da.ts
@@ -12565,8 +12565,8 @@
         <translation>Kunne ikke starte værtens USB-proxytjeneste (VERR_FILE_NOT_FOUND). Tjenesten er muligvis ikke installeret på værtsmaskinen</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBox har ikke rettigheder til at tilgå USB-enheder. Du kan ændre dette ved at tilføje din bruger til gruppen &apos;vboxusers&apos;. Se dokumentationen for yderligere detaljer</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBox har ikke rettigheder til at tilgå USB-enheder. Du kan ændre dette ved at tilføje din bruger til gruppen &apos;wheel&apos;. Se dokumentationen for yderligere detaljer</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_de.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_de.ts
index 14512835..9abd8afc 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_de.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_de.ts
@@ -13900,8 +13900,8 @@ a guest session</source>
         <translation>Der USB-Proxy-Dienst konnte nicht gestarted werden (VERR_FILE_NOT_FOUND). Dieser Dienst ist möglicherweise nicht auf diesem Computer installiert</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBox kann nicht auf USB-Geräte zugreifen. Dieses Problem kann dadurch gelöst werden, indem der aktuelle Nutzer Mitglied der Gruppe &apos;vboxusers&apos; wird. In der Dokumentation finden Sie weitere Erläuterungen</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBox kann nicht auf USB-Geräte zugreifen. Dieses Problem kann dadurch gelöst werden, indem der aktuelle Nutzer Mitglied der Gruppe &apos;wheel&apos; wird. In der Dokumentation finden Sie weitere Erläuterungen</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_el.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_el.ts
index 5823e6ab..2a954efd 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_el.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_el.ts
@@ -13788,8 +13788,8 @@
         <translation>Δεν φορτώθηκε η υπηρεσία διαμεσολάβησης USB του οικοδεσπότη (VERR_FILE_NOT_FOUND). Η υπηρεσία μπορεί να μην έχει εγκατασταθεί στον υπολογιστή</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>Το VirtualBox δεν έχει αυτή τη στιγμή πρόσβαση στις συσκευές USB. Μπορείτε να το αλλάξετε προσθέτοντας τον χρήστη στην ομάδα &apos;vboxusers&apos;. Δείτε το εγχειρίδιο για λεπτομερέστερη εξήγηση</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>Το VirtualBox δεν έχει αυτή τη στιγμή πρόσβαση στις συσκευές USB. Μπορείτε να το αλλάξετε προσθέτοντας τον χρήστη στην ομάδα &apos;wheel&apos;. Δείτε το εγχειρίδιο για λεπτομερέστερη εξήγηση</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_es.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_es.ts
index 8b92199e..ce36fdfe 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_es.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_es.ts
@@ -18839,8 +18839,8 @@ p, li { white-space: pre-wrap; }
         <translation>No se puede cargar el servicio Host USB Proxy (VERR_FILE_NOT_FOUND). El servicio podría no estar instalado en la máquina anfitrión</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBox no tiene permisos para acceder a los dispositivos  USB. Puede cambiar esto agregando su usuario al grupo «vboxusers».  Mire el manual de usuario para una explicación más detallada</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBox no tiene permisos para acceder a los dispositivos  USB. Puede cambiar esto agregando su usuario al grupo «wheel».  Mire el manual de usuario para una explicación más detallada</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_eu.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_eu.ts
index a817c14b..7833937f 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_eu.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_eu.ts
@@ -18349,8 +18349,8 @@
     </message>
     <message>
         <location line="+1"/>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBox-ek ez du sarbiderik USB gailuetara.  Hau zure erabiltzailea &apos;vboxusers&apos; taldera gehituz aldatu dezakezu.  Mesedez ikusi erabiltzaile eskuliburua azalpen xehetuagorako</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBox-ek ez du sarbiderik USB gailuetara.  Hau zure erabiltzailea &apos;wheel&apos; taldera gehituz aldatu dezakezu.  Mesedez ikusi erabiltzaile eskuliburua azalpen xehetuagorako</translation>
     </message>
     <message>
         <location line="+1"/>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fa.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fa.ts
index 9ec92a17..3e8a4ca9 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fa.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fa.ts
@@ -15375,8 +15375,8 @@
         <translation>قادر به بارگیری سرویس پراکسی یو اِس بی میزبان نیست (VERR_FILE_NOT_FOUND). شاید سرویس در رایانه میزبان نصب نشده است</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>ویرچوال باکس اکنون اجازه دسترسی به دستگاه یو ایس بی را ندارد.  این را میتوانید با افزودن کاربر خود به گروه &apos;vboxusers&apos; تغییر دهید.  لطفا برای توضیحات جزئی تر راهنمای کاربری را ببینید</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>ویرچوال باکس اکنون اجازه دسترسی به دستگاه یو ایس بی را ندارد.  این را میتوانید با افزودن کاربر خود به گروه &apos;wheel&apos; تغییر دهید.  لطفا برای توضیحات جزئی تر راهنمای کاربری را ببینید</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fi.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fi.ts
index 3641e416..923b8003 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fi.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fi.ts
@@ -13424,7 +13424,7 @@ p, li { white-space: pre-wrap; }
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fr.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fr.ts
index 6f3ff6ee..52fde048 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fr.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fr.ts
@@ -17151,8 +17151,8 @@
         <translation>Le service de relais de périphériques USB n&apos;a pas pû être activé (erreur VERR_FILE_NOT_FOUND). Il se peut que le service ne soit pas installé sur l&apos;ordinateur hôte</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBox n&apos;est pas autorisé à accéder aux périphériques USB actuellement. Vous pouvez changer ceci en ajoutant votre nom d&apos;utilisateur au groupe «vboxusers». Veuillez consulter le guide de l&apos;utilisateur pour des explications plus détaillées</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBox n&apos;est pas autorisé à accéder aux périphériques USB actuellement. Vous pouvez changer ceci en ajoutant votre nom d&apos;utilisateur au groupe «wheel». Veuillez consulter le guide de l&apos;utilisateur pour des explications plus détaillées</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_gl.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_gl.ts
index bc231b63..81734737 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_gl.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_gl.ts
@@ -13379,7 +13379,7 @@ p, li { white-space: pre-wrap; }
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_he.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_he.ts
index 7d8288dc..57b90b8c 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_he.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_he.ts
@@ -10825,7 +10825,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_hr_HR.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_hr_HR.ts
index c856da53..478e3dad 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_hr_HR.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_hr_HR.ts
@@ -8399,8 +8399,8 @@
         <translation>Nisam uspio učitati Domaćinov USB Posrednički Servis (VERR_FILE_NOT_FOUND). Servis možda nije instaliran na računalo domaćina</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBoxu trenutačno nije dozvoljen pristup USB uređajima.  Možete promijeniti to dodavajući svog korisnika &apos;vboxusers&apos; grupi.  Pogledajte korisnička uputstva za detaljnije objašnjenje</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBoxu trenutačno nije dozvoljen pristup USB uređajima.  Možete promijeniti to dodavajući svog korisnika &apos;wheel&apos; grupi.  Pogledajte korisnička uputstva za detaljnije objašnjenje</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_hu.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_hu.ts
index 60bd9d43..4bcccb9d 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_hu.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_hu.ts
@@ -18306,8 +18306,8 @@ p, li { white-space: pre-wrap; }
         <translation>Nem tölthető be a gazda USB Proxy szolgáltatás (FÁJL_NEM_TALÁLHATÓ). A szolgáltatás talán nincs telepítve a gazda számítógépre</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>A VirtualBox jelenleg nem férhet hozzá az USB eszközökhöz.  Ezen változtathatsz, ha a felhasználód hozzáadod a &apos;vboxusers&apos; csoporthoz.  Részletes magyarázatért nézd meg a Felhasználói kézikönyvet</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>A VirtualBox jelenleg nem férhet hozzá az USB eszközökhöz.  Ezen változtathatsz, ha a felhasználód hozzáadod a &apos;wheel&apos; csoporthoz.  Részletes magyarázatért nézd meg a Felhasználói kézikönyvet</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_id.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_id.ts
index 39345a9d..f1f5949c 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_id.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_id.ts
@@ -17347,8 +17347,8 @@
         <translation>Tak bisa memuat Layanan Proksi USB Host (VERR_FILE_NOT_FOUND). Layanan mungkin tak terinstal pada komputer host</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBox kini tak diijinkan mengakses peranti USB. Anda dapat mengubah ini dengan cara menambahkan pengguna Anda ke grup &apos;vboxusers&apos;.  Silakan lihat manual pengguna untuk penjelasan lebih terinci</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBox kini tak diijinkan mengakses peranti USB. Anda dapat mengubah ini dengan cara menambahkan pengguna Anda ke grup &apos;wheel&apos;.  Silakan lihat manual pengguna untuk penjelasan lebih terinci</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_it.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_it.ts
index 11363ab4..86327587 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_it.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_it.ts
@@ -18966,8 +18966,8 @@ p, li { white-space: pre-wrap; }
         <translation>Impossibile caricare il servizio Host USB Proxy (VERR_FILE_NOT_FOUND). Il servizio potrebbe non essere installato sul computer host</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>Attualmente VirtualBox non ha il permesso di accedere ai dispositivi USB. Puoi cambiare questa situazione aggiungendo il tuo utente al gruppo &apos;vboxusers&apos;. Vedi il manuale utente per una spiegazione più dettagliata</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>Attualmente VirtualBox non ha il permesso di accedere ai dispositivi USB. Puoi cambiare questa situazione aggiungendo il tuo utente al gruppo &apos;wheel&apos;. Vedi il manuale utente per una spiegazione più dettagliata</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ja.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ja.ts
index 065270f9..ab9cc112 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ja.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ja.ts
@@ -19289,8 +19289,8 @@ p, li { white-space: pre-wrap; }
         <translation>USBプロキシーサービスが読み込めません (VERR_FILE_NOT_FOUND)。このサービスはホストコンピューターにインストールされている必要があります</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBoxは現在USBデバイスへのアクセスが許可されていません。ユーザーを&apos;vboxusers&apos;グループに追加してください。詳細な説明はユーザーマニュアルをお読みください</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBoxは現在USBデバイスへのアクセスが許可されていません。ユーザーを&apos;wheel&apos;グループに追加してください。詳細な説明はユーザーマニュアルをお読みください</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ka.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ka.ts
index 08b6aee9..125da936 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ka.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ka.ts
@@ -8365,7 +8365,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_km_KH.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_km_KH.ts
index ca40c4e4..b3ebfc9a 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_km_KH.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_km_KH.ts
@@ -13306,7 +13306,7 @@ medium</comment>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ko.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ko.ts
index 763ec9b7..bab28058 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ko.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ko.ts
@@ -11161,8 +11161,8 @@
         <translation>호스트 USB 프록시 서비스를 불러올 수 없습니다(VERR_FILE_NOT_FOUND). 호스트 컴퓨터에 서비스가 설치되어 있지 않을 수도 있습니다</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBox에서 USB 장치에 접근할 수 없습니다. 현재 사용자를 &apos;vboxusers&apos; 그룹에 추가하십시오. 더 자세한 설명은 사용자 설명서를 참고하십시오</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBox에서 USB 장치에 접근할 수 없습니다. 현재 사용자를 &apos;wheel&apos; 그룹에 추가하십시오. 더 자세한 설명은 사용자 설명서를 참고하십시오</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_lt.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_lt.ts
index f3b54efb..cf8d0572 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_lt.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_lt.ts
@@ -15760,8 +15760,8 @@
         <translation>Nepavyksta įkelti pagr. kompiuterio USB įgaliotojo serverio paslaugos (VERR_FILE_NOT_FOUND). Galbūt ši paslauga neįdiegta pagrindiniame kompiueryje</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBox programai neleidote pasiekti USB įrenginių. Šią parinktį galite pakeisti savo naudotoją priskirdami „vboxusers“ grupei. Daugiau informacijos rasite naudotojo žinyne</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBox programai neleidote pasiekti USB įrenginių. Šią parinktį galite pakeisti savo naudotoją priskirdami „wheel“ grupei. Daugiau informacijos rasite naudotojo žinyne</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_nl.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_nl.ts
index adfd61da..8ad0212f 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_nl.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_nl.ts
@@ -7948,8 +7948,8 @@
         <translation>Kon de Host USB Proxy Service (VERR_FILE_NOT_FOUND) niet laden. De service is mogelijk niet geïnstalleerd op de hostcomputer</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBox wordt momenteel niet toegestaan USB-apparaten te benaderen.  U kunt dit veranderen door uw gebruiker toe te voegen aan de groep &apos;vboxusers&apos;.  Lees de gebruikershandleiding voor een meer gedetailleerde uitleg</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBox wordt momenteel niet toegestaan USB-apparaten te benaderen.  U kunt dit veranderen door uw gebruiker toe te voegen aan de groep &apos;wheel&apos;.  Lees de gebruikershandleiding voor een meer gedetailleerde uitleg</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pl.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pl.ts
index d9f4a55f..85a76f57 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pl.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pl.ts
@@ -16559,8 +16559,8 @@ p, li { white-space: pre-wrap; }
         <translation>Nie można załadować usługi Host USB Proxy (VERR_FILE_NOT_FOUND). Usługa ta być może nie została zainstalowana na komputerze gospodarza</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBox aktualnie nie ma dostępu do urządzeń USB. Można to zmienić, dodając użytkownika do grupy vboxusers&apos;. Proszę zapoznać się z instrukcją obsługi w celu bardziej szczegółowego wyjaśnienia</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBox aktualnie nie ma dostępu do urządzeń USB. Można to zmienić, dodając użytkownika do grupy wheel&apos;. Proszę zapoznać się z instrukcją obsługi w celu bardziej szczegółowego wyjaśnienia</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pt.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pt.ts
index ace78eee..6b755b87 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pt.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pt.ts
@@ -14840,7 +14840,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pt_BR.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pt_BR.ts
index 0913e9e2..7cfde981 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pt_BR.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pt_BR.ts
@@ -18912,8 +18912,8 @@ p, li { white-space: pre-wrap; }
         <translation>Não foi possível carregar o Serviço de Proxy USB do Hospedeiro (VERR_FILE_NOT_FOUND). O serviço pode não estar instalado no computador hospedeiro</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>O VirtualBox não está conseguindo acessar os dispositivos USB.  Você pode alterar este comportamento acrescentando o seu usuário ao grupo &apos;vboxusers&apos;.  Leia o manual do usuário para uma explicação mais detalhada</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>O VirtualBox não está conseguindo acessar os dispositivos USB.  Você pode alterar este comportamento acrescentando o seu usuário ao grupo &apos;wheel&apos;.  Leia o manual do usuário para uma explicação mais detalhada</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ro.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ro.ts
index 332718f8..48e917b1 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ro.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ro.ts
@@ -13535,7 +13535,7 @@ Acest director este folosit, dacă nu este explicit specificat altfel, atunci c
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ru.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ru.ts
index b175c783..919241db 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ru.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ru.ts
@@ -18427,8 +18427,8 @@ p, li { white-space: pre-wrap; }
         <translation>Не удалось загрузить USB Proxy службу хоста (VERR_FILE_NOT_FOUND). Возможно, данная служба не установлена на данном компьютере</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>В данный момент VirtualBox не может использовать устройства USB. Вы можете исправить данную проблему, добавив текущего пользователя в группу &apos;vboxusers&apos;. Для получения более детальных объяснений следуйте указаниям руководства пользователя</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>В данный момент VirtualBox не может использовать устройства USB. Вы можете исправить данную проблему, добавив текущего пользователя в группу &apos;wheel&apos;. Для получения более детальных объяснений следуйте указаниям руководства пользователя</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sk.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sk.ts
index 81324763..c198200a 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sk.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sk.ts
@@ -13881,7 +13881,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sl.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sl.ts
index 4dfc6c7a..15023e3e 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sl.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sl.ts
@@ -15662,8 +15662,8 @@
     </message>
     <message>
         <location line="+1"/>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBox trenutno ne sme dostopati do naprav USB. To lahko spremenite z dodajanjem svojega uporabnika v skupino &apos;vboxusers&apos;. Za bolj podrobno razlago glejte uporabniški priročnik.</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBox trenutno ne sme dostopati do naprav USB. To lahko spremenite z dodajanjem svojega uporabnika v skupino &apos;wheel&apos;. Za bolj podrobno razlago glejte uporabniški priročnik.</translation>
     </message>
     <message>
         <location line="+1"/>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sr.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sr.ts
index e329972e..7887a4d1 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sr.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sr.ts
@@ -14597,8 +14597,8 @@ p, li { white-space: pre-wrap; }
         <translation>Не могу да учитам Host USB Proxy Service (VERR_FILE_NOT_FOUND). Сервис можда није инсталиран на хосту</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBox нема приступа USB уређајима. Можете то да промените тако што додате корисник у групи &apos;vboxusers&apos;.  Видети приручник за више детаље</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBox нема приступа USB уређајима. Можете то да промените тако што додате корисник у групи &apos;wheel&apos;.  Видети приручник за више детаље</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sv.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sv.ts
index ce865778..395cb4ad 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sv.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sv.ts
@@ -16193,8 +16193,8 @@ serial ports</comment>
         <translation>Kunde inte läsa in USB-proxytjänsten för värden (VERR_FILE_NOT_FOUND). Tjänsten kanske inte är installerad på värddatorn</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBox tillåts för närvarande inte att komma åt USB-enheter.  Du kan ändra detta genom att lägga till din användare till gruppen &quot;vboxusers&quot;.  Se användarhandboken för ytterligare information</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBox tillåts för närvarande inte att komma åt USB-enheter.  Du kan ändra detta genom att lägga till din användare till gruppen &quot;wheel&quot;.  Se användarhandboken för ytterligare information</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_th.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_th.ts
index 5c4f3605..37d80f7a 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_th.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_th.ts
@@ -12440,8 +12440,8 @@
         <translation>ไม่สามารถโหลดบริการพรอกซี USB ของโฮสต์ได้  (VERR_FILE_NOT_FOUND) บริการนี้อาจไม่ได้ถูกติดตั้งไว้บนคอมพิวเตอร์โฮสต์</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>เวอร์ชวลบอกซ์ไม่ได้รับอนุญาตให้เข้าถึงอุปกรณ์ USB คุณสามารถแก้ไขได้ด้วยการเพิ่มผู้ใช้ของคุณไปยังกลุ่ม &apos;vboxusers&apos; คำอธิบายโดยละเอียดสามารถดูเพิ่มเติมได้จากคู่มือผู้ใช้</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>เวอร์ชวลบอกซ์ไม่ได้รับอนุญาตให้เข้าถึงอุปกรณ์ USB คุณสามารถแก้ไขได้ด้วยการเพิ่มผู้ใช้ของคุณไปยังกลุ่ม &apos;wheel&apos; คำอธิบายโดยละเอียดสามารถดูเพิ่มเติมได้จากคู่มือผู้ใช้</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_tr.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_tr.ts
index f4b37d8a..ce60702c 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_tr.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_tr.ts
@@ -18301,8 +18301,8 @@ Oturum Panelini kullanın</translation>
         <translation>Anamakine USB Vekil Hizmeti (VERR_FILE_NOT_FOUND) yüklenemedi. Hizmet anamakine bilgisayarına yüklenememiştir</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBox şu anda USB aygıtlara erişime izin vermiyor. Bunu kullanıcınızı &apos;vboxusers&apos; grubuna ekleyerek değiştirebilirsiniz. Lütfen daha fazla ayrıntılı açıklama için kullanım kılavuzuna bakın</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBox şu anda USB aygıtlara erişime izin vermiyor. Bunu kullanıcınızı &apos;wheel&apos; grubuna ekleyerek değiştirebilirsiniz. Lütfen daha fazla ayrıntılı açıklama için kullanım kılavuzuna bakın</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_uk.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_uk.ts
index 4e250fa2..b0355eac 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_uk.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_uk.ts
@@ -16352,8 +16352,8 @@ p, li { white-space: pre-wrap; }
         <translation>Неможливо завантажити головний проксі сервер USB (VERR_FILE_NOT_FOUND). Службу, можливо, не встановили на головному комп&apos;ютері</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBox зараз не дозволяє доступитись до пристроїв USB. Ви моежете змінити це, додавши користувача до групи «vboxusers». Будь ласка, ознайомтесь з користувацьким підручником для докладнішого пояснення</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBox зараз не дозволяє доступитись до пристроїв USB. Ви моежете змінити це, додавши користувача до групи «wheel». Будь ласка, ознайомтесь з користувацьким підручником для докладнішого пояснення</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_xx_YY.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_xx_YY.ts
index 0c8e5ae5..229824f3 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_xx_YY.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_xx_YY.ts
@@ -8351,7 +8351,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_zh_CN.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_zh_CN.ts
index e123ade8..e0e1a1d0 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_zh_CN.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_zh_CN.ts
@@ -16834,8 +16834,8 @@
         <translation>无法载入主机 USB 代理服务 (VERR_FILE_NOT_FOUND)。主机电脑中可能没有安装服务</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBox 当前不允许访问 USB 设备。您可以将您的用户添加到“vboxusers”组来更改以上的问题。请查阅用户手册以获得更多详细的说明</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBox 当前不允许访问 USB 设备。您可以将您的用户添加到“wheel”组来更改以上的问题。请查阅用户手册以获得更多详细的说明</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_zh_TW.ts b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_zh_TW.ts
index 48945ea9..263e5faf 100644
--- a/src/VBox/Frontends/VirtualBox/nls/VirtualBox_zh_TW.ts
+++ b/src/VBox/Frontends/VirtualBox/nls/VirtualBox_zh_TW.ts
@@ -8355,8 +8355,8 @@
         <translation>無法載入主機 USB Proxy 服務  (VERR_FILE_NOT_FOUND)。 服務可能未安裝在主機上</translation>
     </message>
     <message>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>目前不允許 VirtualBox 存取 USB 裝置。 您可以透過將使用者加入到「vboxusers」群組來變更這個設定。 有關更詳細的說明，請參閱使用者手冊</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>目前不允許 VirtualBox 存取 USB 裝置。 您可以透過將使用者加入到「wheel」群組來變更這個設定。 有關更詳細的說明，請參閱使用者手冊</translation>
     </message>
     <message>
         <source>VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the &apos;usbfs&apos; folder and files.  Please see the user manual for a more detailed explanation</source>
diff --git a/src/VBox/Frontends/VirtualBox/src/globals/UIMessageCenter.cpp b/src/VBox/Frontends/VirtualBox/src/globals/UIMessageCenter.cpp
index 6e562a0b..09e42c4b 100644
--- a/src/VBox/Frontends/VirtualBox/src/globals/UIMessageCenter.cpp
+++ b/src/VBox/Frontends/VirtualBox/src/globals/UIMessageCenter.cpp
@@ -2262,7 +2262,7 @@ void UIMessageCenter::prepare()
     /* Translations for Main.
      * Please make sure they corresponds to the strings coming from Main one-by-one symbol! */
     tr("Could not load the Host USB Proxy Service (VERR_FILE_NOT_FOUND). The service might not be installed on the host computer");
-    tr("VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the 'vboxusers' group.  Please see the user manual for a more detailed explanation");
+    tr("VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the 'wheel' group.  Please see the user manual for a more detailed explanation");
     tr("VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the 'usbfs' folder and files.  Please see the user manual for a more detailed explanation");
     tr("The USB Proxy Service has not yet been ported to this host");
     tr("Could not load the Host USB Proxy service");
diff --git a/src/VBox/HostDrivers/linux/loadall.sh b/src/VBox/HostDrivers/linux/loadall.sh
index 7d7d80a0..2a281016 100755
--- a/src/VBox/HostDrivers/linux/loadall.sh
+++ b/src/VBox/HostDrivers/linux/loadall.sh
@@ -36,13 +36,13 @@
 
 TARGET=`readlink -e -- "${0}"` || exit 1 # The GNU-specific way.
 MY_DIR="${TARGET%/[!/]*}"
-MY_GROUPNAME="vboxusers"
+MY_GROUPNAME="wheel"
 
 #
-# vboxusers membership check.
+# wheel membership check.
 #
 if ! (id -Gn | grep -w ${MY_GROUPNAME}); then
-    echo "loadall.sh: you're not a member of vboxusers...";
+    echo "loadall.sh: you're not a member of wheel...";
     # Create the group.
     if ! getent group ${MY_GROUPNAME}; then
         set -e
diff --git a/src/VBox/Installer/linux/VBoxCreateUSBNode.sh b/src/VBox/Installer/linux/VBoxCreateUSBNode.sh
index 6893a6c0..b2e74daa 100755
--- a/src/VBox/Installer/linux/VBoxCreateUSBNode.sh
+++ b/src/VBox/Installer/linux/VBoxCreateUSBNode.sh
@@ -46,7 +46,7 @@ case "$do_remove" in
   then
       exit 0
   fi
-  case "$group" in "") group="vboxusers";; esac
+  case "$group" in "") group="wheel";; esac
   mkdir /dev/vboxusb -m 0750 2>/dev/null
   chown root:$group /dev/vboxusb 2>/dev/null
   mkdir "$devdir" -m 0750 2>/dev/null
diff --git a/src/VBox/Installer/linux/debian/postinst b/src/VBox/Installer/linux/debian/postinst
index 923608f6..13a29970 100755
--- a/src/VBox/Installer/linux/debian/postinst
+++ b/src/VBox/Installer/linux/debian/postinst
@@ -59,9 +59,9 @@ if [ "$1" = "configure" ]; then
 
   # create users groups (disable with INSTALL_NO_GROUP=1 in /etc/default/virtualbox)
   if [ "$INSTALL_NO_GROUP" != "1" ]; then
-    db_input low virtualbox/group-vboxusers || true
+    db_input low virtualbox/group-wheel || true
     db_go || true
-    addgroup --system vboxusers || true
+    addgroup --system wheel || true
   fi
 
   # The starters need to be set-uid-to-root. They drop the privileges before
diff --git a/src/VBox/Installer/linux/debian/templates b/src/VBox/Installer/linux/debian/templates
index 341644b5..ec12350b 100644
--- a/src/VBox/Installer/linux/debian/templates
+++ b/src/VBox/Installer/linux/debian/templates
@@ -1,9 +1,9 @@
-Template: virtualbox/group-vboxusers
+Template: virtualbox/group-wheel
 Type: note
-Description: Creating group 'vboxusers'
+Description: Creating group 'wheel'
  Users of VirtualBox must be members of this group. Host network interfaces
  will be assigned to this group.
-Description-de.UTF-8: Erzeuge Gruppe 'vboxusers'
+Description-de.UTF-8: Erzeuge Gruppe 'wheel'
  Nutzer von VirtualBox müssen Mitglied dieser Gruppe sein. Hostinterfaces
  werden dieser Gruppe zugewiesen.
 
diff --git a/src/VBox/Installer/linux/install.sh b/src/VBox/Installer/linux/install.sh
index d32d9270..772604c8 100755
--- a/src/VBox/Installer/linux/install.sh
+++ b/src/VBox/Installer/linux/install.sh
@@ -51,7 +51,7 @@ CONFIG_DIR="/etc/vbox"
 CONFIG="vbox.cfg"
 CONFIG_FILES="filelist"
 DEFAULT_FILES=`pwd`/deffiles
-GROUPNAME="vboxusers"
+GROUPNAME="wheel"
 INSTALLATION_DIR="_INSTALLATION_DIR_"
 LICENSE_ACCEPTED=""
 PREV_INSTALLATION=""
diff --git a/src/VBox/Installer/linux/rpm/VirtualBox.tmpl.spec b/src/VBox/Installer/linux/rpm/VirtualBox.tmpl.spec
index bab95a70..4994bf53 100644
--- a/src/VBox/Installer/linux/rpm/VirtualBox.tmpl.spec
+++ b/src/VBox/Installer/linux/rpm/VirtualBox.tmpl.spec
@@ -294,9 +294,9 @@ rm -f /etc/vbox/module_not_compiled
 # create users groups (disable with INSTALL_NO_GROUP=1 in /etc/default/virtualbox)
 if [ "$INSTALL_NO_GROUP" != "1" ]; then
   echo
-  echo "Creating group 'vboxusers'. VM users must be member of that group!"
+  echo "Creating group 'wheel'. VM users must be member of that group!"
   echo
-  groupadd -r -f vboxusers 2> /dev/null
+  groupadd -r -f wheel 2> /dev/null
 fi
 
 %if %{?rpm_mdv:1}%{!?rpm_mdv:0}
diff --git a/src/VBox/Installer/linux/vboxdrv.sh b/src/VBox/Installer/linux/vboxdrv.sh
index 5e6842cf..635bb3dc 100755
--- a/src/VBox/Installer/linux/vboxdrv.sh
+++ b/src/VBox/Installer/linux/vboxdrv.sh
@@ -94,7 +94,7 @@ if test -u "${INSTALL_DIR}/VirtualBoxVM" || test '!' -e "${INSTALL_DIR}/VirtualB
     GROUP=root
     DEVICE_MODE=0600
 else
-    GROUP=vboxusers
+    GROUP=wheel
     DEVICE_MODE=0660
 fi
 
@@ -591,7 +591,7 @@ See the documentation for your Linux distribution." console
     # of USB access.  The USB code checks for the existence of that path.
     if grep -q usb_device /proc/devices; then
         mkdir -p -m 0750 /dev/vboxusb 2>/dev/null
-        chown root:vboxusers /dev/vboxusb 2>/dev/null
+        chown root:wheel /dev/vboxusb 2>/dev/null
     fi
     # Remove any kernel modules left over from previously installed kernels.
     cleanup only_old
diff --git a/src/VBox/Installer/solaris/vbox-ips.mog b/src/VBox/Installer/solaris/vbox-ips.mog
index b5bacd49..c6bd27ec 100644
--- a/src/VBox/Installer/solaris/vbox-ips.mog
+++ b/src/VBox/Installer/solaris/vbox-ips.mog
@@ -7,7 +7,7 @@ set name=variant.arch value=@UNAME_P@
 set name=variant.opensolaris.zone value=global value=nonglobal
 set name=info.classification value=org.opensolaris.category.2008:System/Virtualization
 license LICENSE license="GPLv2 (virtualbox)"
-group groupname=vboxusers gid=86
+group groupname=wheel gid=86
 # The data-xkb was renamed to xkeyboard-config in S11.4, so adjust this when
 # we have moved our builds away from S11.3.
 depend fmri=pkg:/x11/keyboard/data-xkb type=require
@@ -62,7 +62,7 @@ $(HARDENED_ONLY)<transform file path=opt/VirtualBox/amd64/(VirtualBoxVM|VBoxHead
 <transform file path=platform/i86pc/kernel/drv/amd64/vboxdrv$ -> emit driver name=vboxdrv perms="* 0600 root sys" perms="vboxdrvu 0666 root sys" devlink=type=ddi_pseudo;name=vboxdrv;minor=vboxdrv\t\D devlink=type=ddi_pseudo;name=vboxdrv;minor=vboxdrvu\t\M0>
 <transform file path=platform/i86pc/kernel/drv/amd64/vboxnet$ -> emit driver name=vboxnet>
 <transform file path=platform/i86pc/kernel/drv/amd64/vboxbow$ -> emit driver name=vboxbow>
-<transform file path=platform/i86pc/kernel/drv/amd64/vboxusbmon$ -> emit driver name=vboxusbmon perms="* 0660 root vboxusers" devlink=type=ddi_pseudo;name=vboxusbmon\t\D>
+<transform file path=platform/i86pc/kernel/drv/amd64/vboxusbmon$ -> emit driver name=vboxusbmon perms="* 0660 root wheel" devlink=type=ddi_pseudo;name=vboxusbmon\t\D>
 <transform file path=platform/i86pc/kernel/drv/amd64/vboxusb$ -> emit driver name=vboxusb>
 <transform file path=platform/i86pc/kernel/drv/.*$ -> set group sys>
 
diff --git a/src/VBox/Main/nls/VirtualBoxAPI_ru.ts b/src/VBox/Main/nls/VirtualBoxAPI_ru.ts
index 252b512f..3a0e000b 100644
--- a/src/VBox/Main/nls/VirtualBoxAPI_ru.ts
+++ b/src/VBox/Main/nls/VirtualBoxAPI_ru.ts
@@ -5768,8 +5768,8 @@ Guest offers: %s</source>
     </message>
     <message>
         <location filename="../src-server/HostImpl.cpp" line="3621"/>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>В настоящее время VirtualBox не имеет доступа к USB устройствам. Вы можете изменить это путем добавления пользователя в группу &apos;vboxusers&apos;. Пожалуйста, обратитесь к руководству пользователя за более подробными объяснениями</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>В настоящее время VirtualBox не имеет доступа к USB устройствам. Вы можете изменить это путем добавления пользователя в группу &apos;wheel&apos;. Пожалуйста, обратитесь к руководству пользователя за более подробными объяснениями</translation>
     </message>
     <message>
         <location filename="../src-server/HostImpl.cpp" line="3624"/>
diff --git a/src/VBox/Main/nls/VirtualBoxAPI_tr.ts b/src/VBox/Main/nls/VirtualBoxAPI_tr.ts
index de786046..dc6a7c14 100644
--- a/src/VBox/Main/nls/VirtualBoxAPI_tr.ts
+++ b/src/VBox/Main/nls/VirtualBoxAPI_tr.ts
@@ -5714,8 +5714,8 @@ Guest offers: %s</source>
     </message>
     <message>
         <location filename="../src-server/HostImpl.cpp" line="3621"/>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
-        <translation>VirtualBox&apos;ın şu anda USB aygıtlara erişmesine izin verilmiyor. Kullanıcınızı &apos;vboxusers&apos; grubuna ekleyerek bunu değiştirebilirsiniz. Daha ayrıntılı bir açıklama için lütfen kullanım kılavuzuna bakın</translation>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <translation>VirtualBox&apos;ın şu anda USB aygıtlara erişmesine izin verilmiyor. Kullanıcınızı &apos;wheel&apos; grubuna ekleyerek bunu değiştirebilirsiniz. Daha ayrıntılı bir açıklama için lütfen kullanım kılavuzuna bakın</translation>
     </message>
     <message>
         <location filename="../src-server/HostImpl.cpp" line="3624"/>
diff --git a/src/VBox/Main/nls/VirtualBoxAPI_xx_YY.ts b/src/VBox/Main/nls/VirtualBoxAPI_xx_YY.ts
index e523d377..22e1a526 100644
--- a/src/VBox/Main/nls/VirtualBoxAPI_xx_YY.ts
+++ b/src/VBox/Main/nls/VirtualBoxAPI_xx_YY.ts
@@ -5545,7 +5545,7 @@ Guest offers: %s</source>
     </message>
     <message>
         <location filename="../src-server/HostImpl.cpp" line="3621"/>
-        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;vboxusers&apos; group.  Please see the user manual for a more detailed explanation</source>
+        <source>VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the &apos;wheel&apos; group.  Please see the user manual for a more detailed explanation</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/src/VBox/Main/src-client/ConsoleImpl2.cpp b/src/VBox/Main/src-client/ConsoleImpl2.cpp
index 3576c493..bdd963f1 100644
--- a/src/VBox/Main/src-client/ConsoleImpl2.cpp
+++ b/src/VBox/Main/src-client/ConsoleImpl2.cpp
@@ -6163,7 +6163,7 @@ int Console::i_configNetwork(const char *pszDevice,
                                                              "Failed to open '/dev/%s' for read/write access.  Please check the "
                                                              "permissions of that node, and that the net.link.tap.user_open "
                                                              "sysctl is set.  Either run 'chmod 0666 /dev/%s' or change the "
-                                                             "group of that node to vboxusers and make yourself a member of "
+                                                             "group of that node to wheel and make yourself a member of "
                                                              "that group.  Make sure that these changes are permanent."),
                                                              pszBridgedIfName, pszBridgedIfName);
                             default:
diff --git a/src/VBox/Main/src-server/HostImpl.cpp b/src/VBox/Main/src-server/HostImpl.cpp
index 2ae1c04a..8616a882 100644
--- a/src/VBox/Main/src-server/HostImpl.cpp
+++ b/src/VBox/Main/src-server/HostImpl.cpp
@@ -3618,7 +3618,7 @@ HRESULT Host::i_checkUSBProxyService()
                                   tr("Could not load the Host USB Proxy Service (VERR_FILE_NOT_FOUND).  The service might not be installed on the host computer"));
             case VERR_VUSB_USB_DEVICE_PERMISSION:
                 return setWarning(E_FAIL,
-                                  tr("VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the 'vboxusers' group.  Please see the user manual for a more detailed explanation"));
+                                  tr("VirtualBox is not currently allowed to access USB devices.  You can change this by adding your user to the 'wheel' group.  Please see the user manual for a more detailed explanation"));
             case VERR_VUSB_USBFS_PERMISSION:
                 return setWarning(E_FAIL,
                                   tr("VirtualBox is not currently allowed to access USB devices.  You can change this by allowing your user to access the 'usbfs' folder and files.  Please see the user manual for a more detailed explanation"));
-- 
2.44.0

